taps_to_cases/runner/src/main/java/com/microsoft/hydralab/t2c/runner/T2CAppiumUtils.java

```diff
@@ -57,9 +57,10 @@ public static void doAction(@NotNull BaseDriverController driver, @NotNull Actio
         } catch (Exception e) {
             e.printStackTrace();
             int index = actionInfo.getId();
-            logger.error("doAction at step " + index + "with exception: " + e.getMessage());
+            String description = actionInfo.getDescription();
+            logger.error("doAction at " + index + ", description: " + description + ", with exception: " + e.getMessage());
             if (!isOption) {
-                throw new IllegalStateException("Failed at step " + index + ": " + e.getMessage()
+                throw new IllegalStateException("Failed at " + index + ", description: " + description + ", " + e.getMessage()
```

Comment:
How about hide "Ddescription: " when description is empty.

Reply:
It's bit of confusing if someone else see the error message when there is nothing about description, I would suggest we keep the same structure of the error message.

Reply:
It's ok